gpu-operator/gpu-operator-kubevirt.rst

```diff
@@ -144,52 +172,59 @@ Install the GPU Operator, enabling ``sandboxWorkloads``:
+#. Create an ImagePullSecret for accessing the NVIDIA vGPU Manager image:
 
-.. code-block:: console
+   .. code-block:: console
 
-    $ kubectl create secret docker-registry ${REGISTRY_SECRET_NAME} \
-      --docker-server=${PRIVATE_REGISTRY} --docker-username=<username> \
-      --docker-password=<password> \
-      --docker-email=<email-id> -n gpu-operator
+      $ kubectl create secret docker-registry ${REGISTRY_SECRET_NAME} \
+         --docker-server=${PRIVATE_REGISTRY} --docker-username=<username> \
+         --docker-password=<password> \
+         --docker-email=<email-id> -n gpu-operator
 
-Install the GPU Operator with ``sandboxWorkloads`` and ``vgpuManager`` enabled and specify the NVIDIA vGPU Manager image built previously:
+#. Install the GPU Operator with ``sandboxWorkloads`` and ``vgpuManager`` enabled and specify the NVIDIA vGPU Manager image built previously:
 
-.. code-block:: console
+   .. code-block:: console
 
-   $ helm install --wait --generate-name \
-         -n gpu-operator --create-namespace \
-         nvidia/gpu-operator \
-         --version=${version} \
-         --set sandboxWorkloads.enabled=true \
-         --set vgpuManager.enabled=true \
-         --set vgpuManager.repository=<path to private repository> \
-         --set vgpuManager.image=vgpu-manager \
-         --set vgpuManager.version=<driver version> \
-         --set vgpuManager.imagePullSecrets={${REGISTRY_SECRET_NAME}}
+      $ helm install --wait --generate-name \
+            -n gpu-operator --create-namespace \
+            nvidia/gpu-operator \
+            --version=${version} \
+            --set sandboxWorkloads.enabled=true \
+            --set vgpuManager.enabled=true \
+            --set vgpuManager.repository=<path to private repository> \
+            --set vgpuManager.image=vgpu-manager \
+            --set vgpuManager.version=<driver version> \
+            --set vgpuManager.imagePullSecrets={${REGISTRY_SECRET_NAME}}
 
 The vGPU Device Manager, deployed by the GPU Operator, automatically creates vGPU devices which can be assigned to KubeVirt virtual machines.
 Without additional configuration, the GPU Operator creates a default set of devices on all GPUs.
 To learn more about how the vGPU Device Manager and configure which types of vGPU devices get created in your cluster, refer to :ref:`vGPU Device Configuration<vgpu-device-configuration>`.
 
 Add GPU resources to KubeVirt CR
---------------------------------
+-------------------------------------
+Follow one of the below subsections for adding GPU resources to the KubeVirt CR, depending on whether you plan to use NVIDIA vGPU or not.
+
+.. _add-vgpu-resources-to-kubevirt-cr:
+
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+Add vGPU resources to KubeVirt CR
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 Update the KubeVirt custom resource so that all GPU and vGPU devices in your cluster are permitted and can be assigned to virtual machines.
```

Comment:
```suggestion
Update the KubeVirt custom resource so that all vGPU devices in your cluster are permitted and can be assigned to virtual machines.
```